README.md

```diff
@@ -10,6 +10,132 @@
+
+- [Linux](https://wiki.freecad.org/Compile_on_Linux)
+- [Windows](https://wiki.freecad.org/Compile_on_Windows)
+- [macOS](https://wiki.freecad.org/Compile_on_MacOS)
+- [Cygwin](https://wiki.freecad.org/Compile_on_Cygwin)
+- [MinGW](https://wiki.freecad.org/Compile_on_MinGW)
+
+Reporting Issues
+---------
+
+To report an issue please:
+
+- First post to forum to verify the issue; 
+- Link forum thread to bug tracker ticket and vice-a-versa; 
+- Use the most updated stable or development versions of FreeCAD; 
+- Post version info from eg. `Help > About FreeCAD > Copy to clipboard`; 
+- Post a Step-By-Step explanation on how to recreate the issue; 
+- Upload an example file to demonstrate problem. 
+
+For more detail see:
+
+- [Bug Tracker](https://github.com/FreeCAD/FreeCAD/issues)
+- [Reporting Issues and Requesting Features](https://github.com/FreeCAD/FreeCAD/issues/new/choose)
+- [Contributing](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md)
+- [Help Forum](https://forum.freecad.org/viewforum.php?f=3)
+- [Developers Handbook](https://freecad.github.io/DevelopersHandbook/)
+
+The [FPA](https://fpa.freecad.org) offers developers the opportunity
+to apply for a grant to work on projects of their choosing. Check
+[jobs and funding](https://blog.freecad.org/jobs/) to know more.
+
+Usage & Getting help
+--------------------
+
+The FreeCAD wiki contains documentation on 
+general FreeCAD usage, Python scripting, and development. These 
+pages might help you get started:
+
+- [Getting started](https://wiki.freecad.org/Getting_started)
+- [Features list](https://wiki.freecad.org/Feature_list)
+- [Frequent questions](https://wiki.freecad.org/FAQ/en)
+- [Workbenches](https://wiki.freecad.org/Workbenches)
+- [Scripting](https://wiki.freecad.org/Power_users_hub)
+- [Development](https://wiki.freecad.org/Developer_hub)
+
+The [FreeCAD forum](https://forum.freecad.org) is also a great place
+to find help and solve specific problems you might encounter when
+learning to use FreeCAD.
+
+Tutorials
+---------
+
+Check out our [tutorials](tutorials/index.md) to get started with FreeCAD. The tutorials cover a range of topics from beginner to advanced levels, helping you to master FreeCAD.
+
+<p>This project receives generous infrastructure support from
+  <a href="https://www.digitalocean.com/">
+    <img src="https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/SVG/DO_Logo_horizontal_blue.svg" width="91px">
+  </a> and <a href="https://www.kipro-pcb.com/">KiCad Services Corp.</a>
+</p>
+
+<!-- <a href="https://freecad.org"><img src="/src/Gui/Icons/freecad.svg" height="100px" width="100px"></a>
+
+### Your own 3D parametric modeler
+
+[Website](https://www.freecad.org) • 
+[Documentation](https://wiki.freecad.org) •
+[Forum](https://forum.freecad.org/) •
+[Bug tracker](https://github.com/FreeCAD/FreeCAD/issues) •
+[Git repository](https://github.com/FreeCAD/FreeCAD) •
+[Blog](https://blog.freecad.org)
```

Comment:
Why are their asterisks at the end of the line for all of these ? Why is this section commented out ?

Reply:
that is the old readme file which I've just commented out